src/sdk/sell.ts

```diff
@@ -392,6 +395,7 @@ export async function getMeta(
         GDXAmount: inputAmount.lessThan(GDXBalance) ? inputAmount : GDXBalance,
 
         priceImpact,
+        priceImpactPCT,
```

Comment:
that doesnt make sense priceImpact and priceImpactPCT are the same. you should cast priceImpact at the point in code where it is needed

Reply:
done